lib/kubernetes-deploy/discoverable_resource.rb

```diff
@@ -0,0 +1,171 @@
+# frozen_string_literal: true
+require 'kubernetes-deploy/kubernetes_resource'
+require 'kubernetes-deploy/kubeclient_builder'
+require "active_record"
```

Comment:
Let's require only the piece we need, or if that isn't possible, copy the one method. I don't want to pull in all of AR.